src/legacy/Routes.tsx

```diff
@@ -109,11 +110,11 @@ class RouteApp extends React.Component<any, any> {
     this.rpc = rpc
 
     // Auto refresh every 10s
-    this.rpcRefreshIntervalId = window.setInterval(() => {
-      if (this.state.rpcConfig.username) {
-        rpc.refresh()
-      }
-    }, 10000)
+    // this.rpcRefreshIntervalId = window.setInterval(() => {
+    //   if (this.state.rpcConfig.username) {
+    //     rpc.refresh()
+    //   }
+    // }, 10000)
```

Comment:
@bulatsalyakhovdev can you use `if (!app.isPackaged)`? Like this https://github.com/pastelnetwork/pastel-electron-wallet/blob/new_development/src/main.ts#L88

Reply:
Hmm, this was not supposed to go on the commit. Is that the way to prevent it to happen on `new_development`?